article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -146,89 +159,106 @@ of the jacobian derivation see supplementary_notebook_2.ipynb). As an expansion
-b-value of $0 s.mm^{-2}$ and 578 volumes diffusion weighted images acquired along 16 diffusion gradient directions
+Similarly to the original article, the procedures are also tested using in vivo human 
+brain data [@valabregue2015], that can be automatically
+downloaded by Dipy's functions.
+
+stuff here
+
+The original dataset consisted of 74 volumes of images acquired for a
+b-value of $0 s.mm^{-2}$ and 578 diffusion weighted images acquired along 16 diffusion gradient directions
 for b-values of 200 and 400 $s.mm^{-2}$ and along 182 diffusion gradient directions for b-values
 of 1000, 2000 and 3000 $s.mm^{-2}$. In this study, only the data for b-values up to $2000 $s.mm^{-2}$
-are used to decrease the impact of non-Gaussian diffusion effects which are not
+are used to decrease the impact of non-Gaussian diffusion effects, which are not
 taken into account by the free water elimination model [@Hoy2014-lk]. In addition to the free water
 elimination model, we also process the data using the standard DTI tensor model for comparison purposes
 (this model is already implemented in Dipy).
 
 # Results
 
-The results from the Monte Carlo simulations are shown in Figure @fig:simulations. Similarly to what is reported
+The results from the Monte Carlo simulations are shown in Figure @fig:simulations. AS reported similarly
 in the original article, FA values estimated using the free water elimination model match the tissue's ground truth
-values for free water volume fractions $f$ ranging around 0 to 0.7 (top panel of
+values for free water volume fractions $f$ ranging from 0 to 0.7 (top panel of
 Figure @fig:simulations). However, FA values seem to be overestimated for higher volume fractions. This bias is more
-prominent for lower FA values in which overestimations are visible from lower free water volume
+prominent for lower FA values, in which overestimation is visible from lower free water volume
 fractions. The lower panels of Figure @fig:simulations suggest that the free water elimination model produce
 accurate free water volume fraction for the full range of volume fraction ground truth values. All the features observed
 here are consistent with Fig.5 of the original article.
 
-![Fractional Anisotropy (FA) and free water volume fraction ($f$) estimates obtained with the from the Monte Carlo simulations
-using the free water elimination fitting procedures. The top panel shows the FA median and intra-quartil range
-for the five different FA ground truth levels and plotted as function of the ground truth water volume fraction.
-The bottom panels show the estimated volume fraction $f$ median and intra-quartil range as function of its ground truth values
+![Fractional Anisotropy (FA) and free water volume fraction ($f$) estimates obtained with Monte Carlo simulations
+using the free water elimination fitting procedures. The top panel shows the FA median and interquartile range
```

Comment:
unless you meant the intra-quartile range, much more common appellation for Q3-Q1